R/root.R

```diff
@@ -140,7 +139,7 @@ empty_config_contents <- function() {
 ## * also check that the outpack and orderly path are compatibible
 ##   (this is actually quite hard to get right, but should be done
 ##   before anything is created I think)
-root_open <- function(path, locate, require_orderly = FALSE, call = NULL) {
+root_open <- function(path, require_orderly = FALSE, call = parent.frame()) {
   if (is.null(call)) {
```

Comment:
Can we remove this line? Presumably no where calls this with an explicit `call = NULL`.